package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/dex-contracts",
-  "version": "0.4.2",
+  "version": "0.4.3",
```

Comment:
sneaky unrelated change ;)

Reply:
Since really the only thing we are changing here is the deployment I was hoping to do the version bump on one go, but I can take it back and do it in a separate PR.